_layouts/post.html

```diff
@@ -1,7 +1,55 @@
 ---
 layout: default
 ---
-<h2>{{ page.title }}</h2>
-<time datetime="{{ page.date }}">Published {{ page.date | date: "%F" }}</time> | <time datetime="{{ page.date }}">Last edited {{ page.date | date: "%F" }}</time>
+<!-- page title -->
+<section class="page-title-alt bg-primary position-relative">
+    <div class="container">
+      <div class="row">
+        <div class="col-12">
+          <h1 class="text-white font-tertiary">{{ page.title }}</h1>
+        </div>
+      </div>
+    </div>
+  </section>
+  <!-- /page title -->
 
-{{ content }}
+<section class="mt-3">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12">
+        <p class="font-secondary">Published on {{ page.date | date: '%b %d, %Y' }} by <span class="text-primary">{{ page.author }}</span
+            class="text-primary"> on {% for tag in page.tags %}<span>{{ tag }} </span>{% endfor %}</p>
```

Comment:
I think we can omit tags from the view. Don't think we're going to use them much. :)

Reply:
Fixed :)